### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**
/cc @lundibundi @nechaido @belochub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,36 +14,139 @@ Metadata Schema and Interface Definition Language (IDL)
...
+        - `schema`: `<Schema>`
+        - `instance`: `<any>`
+        - `options`: [`<Object>`][object]
+        - `Returns:`: [`<Error[]>`][error]
+      - `creator`: [`<Function>`][function] optional
+        - `ms`: [`<Metaschema>`][metaschema]
+        - `schema`: `<Schema>`
+        - `args`: `<any>`
+        - `options`: [`<Object>`][object]
+        - `Returns:`: `[ `[`<Error[]>`][error]`, ``<any>`` ]`
```
Comment (lundibundi) [inline README.md]: ``` ``<any>`` ``` this looks like an error.

File: README.md [hunk 2]
```diff
@@ -13,36 +14,143 @@ Metadata Schema and Interface Definition Language (IDL)
...
+
+- `name`: [`<string>`][string] schema name
+- `source`: [`<string>`][string] schema source
+- `options`: [`<Object>`][object]
+  - `context`: [`<Object>`][object] object to be assigned to global during
+    loading
+  - `decorators`: [`<Object>`][object] decorators available during schema
+    processing
+  - `localDecorators`: [`<Object>`][object] type specific decorators available
+    during
```
Comment (nechaido) [inline README.md]: This should be:
```suggestion
    during `schema` processing
```

File: README.md [hunk 3]
```diff
@@ -13,36 +14,142 @@ Metadata Schema and Interface Definition Language (IDL)
...
+  - `resolve`: [`<Function>`][function]
+    - `ms`: [`<Metaschema>`][metaschema]
+    - `type`: [`<string>`][string]
+    - `name`: [`<string>`][string]
+    - `Returns:`: `<Schema>`
+  - `processors`: [`<Object>`][object]
+    - `[type]`: [`<Object>`][object]
+      - `preprocess`: [`<Function[]>`][function] optional
+        - `schema`: `<Schema>`
+        - `Returns:`: [`<Error[]>`][error]
```
Comment (lundibundi) [inline README.md]: I assume this is waiting for https://github.com/metarhia/metadoc/pull/46?

Comment (SemenchenkoVitaliy) [reply README.md]: This and also https://github.com/metarhia/globalstorage/pull/305.

**Pull request merged**